cmd/odo/odo.go

```diff
@@ -14,9 +12,38 @@ import (
 	"github.com/redhat-developer/odo/pkg/odo/util/completion"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
 )
 
+const pluginSuffix = ".odo.plugin"
+
+var plugins = make(map[string]plugin, 10)
```

Comment:
I guess this is just for PoC. Or we can't have more than 10 plugins? :-)

Reply:
This is the initial capacity, isn't it? Maps are growing on demand past this initial capacity, no?